File: server/php/src/Infrastructure/Mappers/PaymentDetails/ReattemptInstructionsConditionsMapper.php
<?php

namespace OnlinePayments\ExampleApp\Infrastructure\Mappers\PaymentDetails;

use OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails\ReattemptInstructionsConditions as ReattemptInstructionsConditionsDto;
use OnlinePayments\Sdk\Domain\ReattemptInstructionsConditions as ReattemptInstructionsConditionsSdk;

final class ReattemptInstructionsConditionsMapper
{
    public static function mapFromSdkResponse(?ReattemptInstructionsConditionsSdk $response): ?ReattemptInstructionsConditionsDto
    {
        if ($response === null) {
            return null;
        }

        $dto = new ReattemptInstructionsConditionsDto();
        $dto->maxAttempts = $response?->maxAttempts ?? null;
        $dto->maxDelay = $response?->maxDelay ?? null;

        return $dto;
    }
}

File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/PaymentDetails/ReattemptInstructionsConditionsMapper.cs
using ReattemptInstructionsConditionsSdk = OnlinePayments.Sdk.Domain.ReattemptInstructionsConditions;
using ReattemptInstructionsConditionsDto = Business.Domain.Payments.PaymentDetails.ReattemptInstructionsConditions;

namespace Infrastructure.Mappers.PaymentDetails;

public static class ReattemptInstructionsConditionsMapper
{
    public static ReattemptInstructionsConditionsDto Map(ReattemptInstructionsConditionsSdk? response)
    {
        return new()
        {
            MaxAttempts = response?.MaxAttempts,
            MaxDelay = response?.MaxDelay
        };
    }
}